_data/sites.json

```
@@ -4740,8 +4729,10 @@
 
     {
         "name": "MixCloud",
-        "url": "http://www.mixcloud.com/myaccount/account/",
+        "url": "http://mixcloud.com/settings/delete-account",
```

It looks like this is the URL which is redirected to?
```suggestion
        "url": "https://www.mixcloud.com/settings/delete-account/",
```